tests/helpers/utility-methods.js

```diff
@@ -129,7 +130,13 @@ export function containerSetup(application, serializerType) {
     let store = application.lookup('service:store');
 
     let adapterType = serializerType === '-json' ? '-rest' : serializerType;
-    let adapter = application.lookup('adapter:' + adapterType);
+    let adapter;
```

Comment:
These are test-only changes but it appears that doing the following out-of-the-box works. I need to register the RESTAdapter similar to the other adapters a few lines above.

```
application.lookup('adapter:-rest');
```